deploy/clowdapp.yaml

```diff
@@ -673,9 +673,16 @@ objects:
       secretName: ${FLOORIST_BUCKET_SECRET_NAME}
     suspend: ${{FLOORIST_SUSPEND}}
     logLevel: ${FLOORIST_LOGLEVEL}
+    resources:
+      limits:
+        cpu: ${{FLOORIST_CPU_LIMIT}}
```

Comment:
you have the cpu limit / request, probably create this variable and put there some usual cpu limit as used in other pods